patches/tModLoader/Terraria/Player.cs.patch

```diff
@@ -4180,6 +4180,15 @@
  			int whoAmI = base.whoAmI;
  			bool flag = true;
  			int num = (int)((float)Main.mouseX + Main.screenPosition.X) / 16;
+@@ -36421,6 +_,8 @@
+ 			if (baitTypeUsed == 2673)
+ 				flag = true;
+ 
++			bool? forced = PlayerLoader.CanConsumeBait(this, item) ?? ItemLoader.CanConsumeBait(this, item);
```

Comment:
You might need to use `CombinedHooks` here, because you need to combine the precedence of the two results.

Reply:
This one I had decided that player-based decisions take precedence over item decisions, for scope reasons. But I can change it to always have true as precedence for every single case.